packages/mdc-select/foundation.js

```diff
@@ -85,14 +94,17 @@ export default class MDCSelectFoundation extends MDCFoundation {
     this.displayViaKeyboardHandler_ = (evt) => this.handleDisplayViaKeyboard_(evt);
     this.selectionHandler_ = ({detail}) => {
       const {index} = detail;
-      this.close_();
+      const removeFloatingLabelClass = false;
```

Comment:
It is entirely for code style. Saves a future dev from having to traverse the file to see the effect passing `true/false` would have on the `close_()` method. I will make the name change.